vm/src/stdlib/os.rs

```diff
@@ -1195,6 +1197,50 @@ mod _os {
         }
     }
 
+    #[derive(FromArgs)]
+    struct CopyFileRangeArgs {
+        #[pyarg(positional)]
+        src: i32,
+        #[pyarg(positional)]
+        dst: i32,
+        #[pyarg(positional)]
+        count: i64,
+        #[pyarg(any, default)]
+        offset_src: Option<Offset>,
+        #[pyarg(any, default)]
+        offset_dst: Option<Offset>,
+    }
+
+    #[cfg(target_os = "linux")]
+    #[pyfunction]
+    fn copy_file_range(args: CopyFileRangeArgs, vm: &VirtualMachine) -> PyResult<usize> {
+        let p_offset_src = args.offset_src.as_ref().map_or_else(std::ptr::null, |x| x);
+        let p_offset_dst = args.offset_dst.as_ref().map_or_else(std::ptr::null, |x| x);
+        let count: usize = args
+            .count
+            .try_into()
+            .map_err(|_| vm.new_value_error("count should > 0".to_string()))?;
```

Comment:
`>= 0` I think, since it just can't be negative

Reply:
Yeah, but `args.count` is type `isize` and `count` is type `usize` so the `try_into()` will actually get `>=0` or it will be error.

Reply:
Right, I was saying the error message says `>` but it's actually `>=`

Reply:
Oh this is great catch! Will fix it.